src/patternfly/components/List/list.scss

```diff
@@ -1,11 +1,29 @@
 .pf-c-list {
+  // list
   --pf-c-list--PaddingLeft: var(--pf-global--spacer--lg);
   --pf-c-list--nested--MarginTop: var(--pf-global--spacer--sm);
   --pf-c-list--nested--MarginLeft: var(--pf-global--spacer--sm);
   --pf-c-list--ul--ListStyle: var(--pf-global--ListStyle);
   --pf-c-list--li--MarginTop: var(--pf-global--spacer--sm);
+
+  // inline
   --pf-c-list--m-inline--li--MarginRight: var(--pf-global--spacer--lg);
 
+  // bordered
+  --pf-c-list--m-bordered--li--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-list--m-bordered--li--BorderBottomColor: var(--pf-global--BorderColor--100);
+  --pf-c-list--m-bordered--li--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+
+  // icon
+  --pf-c-list__item-icon--MinWidth: var(--pf-global--FontSize--xs);
```

Comment:
The min-width should match whatever the font-size is so that this fits a square icon or one that is more narrow than it is wide. All of the icons in react are squares, so that is what we're targeting mainly. If a user has icons that are wider than they are tall, they will need to update `--pf-c-list__item-icon--MinWidth` on the list.